includes/snmp.inc.php

```diff
@@ -246,6 +246,7 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 
     $cmd = gen_snmpget_cmd($device, $oid, $options, $mib, $mibdir);
     $data = trim(external_exec($cmd));
+    $data = str_replace('"', '', $data);
```

Comment:
Do not str_replace, that will remove all quotes from the string.  You only need to remove the outside ones. `trim($data, '"')`

Reply:
The same is in the snmp_walk function